warehouse/templates/email/removed-as-collaborator.body.txt

```diff
@@ -0,0 +1,6 @@
+Your role in the {{ project }} project has been removed by {{ submitter }}.
+
+You are no longer an {{ role }}.
```

Comment:
I think we might need to reword this, because it could either be "a maintainer" or "an owner".

It would also be nice to add some kind of text explaining what the role is.

e.g.

"You are now a project owner.  This means that you may add other collaborators, upload releases for the project, delete files and releases, or delete the entire project."

Or

"You are now a project maintainer. This means you may upload releases for the project. You cannot add collaborators, delete files, delete releases or delete the project.

see https://github.com/pypa/warehouse/issues/3157 for reference.

Reply:
Since this particular email is for when they are removed from the role, I will phrase it like "You are no longer a(n) owner/maintainer. You can no longer ...".
Does that sounds ok?

Reply:
Ah, so it is.  Sorry, I misread :)

Maybe "you are no longer a project owner/maintainer" to avoid the "a(n)" issue? wdyt?